.github/workflows/test-release.yml

```
@@ -0,0 +1,130 @@
+          echo "Release helper script test completed"
+
+      - name: Validate semantic-release config
+        run: |
+          echo "Validating semantic-release configuration..."
+          npx semantic-release --dry-run --no-ci || true
+          echo "Semantic-release config validation completed"
+
+      - name: Test release scripts
+        run: |
+          echo "Testing release scripts..."
+          
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)